binding.gyp

```diff
@@ -51,7 +52,6 @@
       ],
 
       'defines': [
-        'NAPI_VERSION=3',
```

Comment:
Doesn't removing this cause issues?

Reply:
I don't think so, as now <node_version.h> is included, which defines NAPI_VERSION...

If I didn't remove this, I got a compilation error that NAPI_VERSION was being redefined.